Tests/StreamChatUITests/SnapshotTests/ChatChannel/ChatChannelVC_Tests.swift

```diff
@@ -652,6 +652,10 @@ final class ChatChannelVC_Tests: XCTestCase {
 
         vc.components.isMessageEditedLabelEnabled = true
 
+        // Also update the default because in snapshot tests, message cells are created before they are in the responder chain
+        defer { Components.default.isMessageEditedLabelEnabled = false }
+        Components.default.isMessageEditedLabelEnabled = true
```

Comment:
Why is this not a problem for the other tests tho? 🤔

Reply:
Depends on the timing: when view is added to the view hierarchy vs when the modified property is accessed. Can't figure out another way how to get this test working. It worked before because ChatMessageContentView.updateContent was called twice (at the second time, the view was part of the view hierarchy) but now it is only once. And really hard to change the timing of the first because it is triggered from the `_View` base view everyone is using.